__tests__/users.test.js

```diff
@@ -0,0 +1,25 @@
+const request = require('supertest');
+const app = require('../app')
+const db = require('../db/connection');
+
+
+
+afterAll(()=>db.end());
+
+//task 10
+describe('/api/users',()=>{
+    test('200 : get all users',()=>{
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body})=>{
+            body.users.forEach((user)=>{
```

Comment:
please add a test for the length of the array before using a forEach. If you are receiving an empty array from the request then all the tests inside the array will be skipped but the length test outside will still run